test/functional/mempool_ephemeral_dust.py

```diff
@@ -323,49 +331,68 @@ def test_reorgs(self):
 
         # Get dusty tx mined, then check that it makes it back into mempool on reorg
         # due to bypass_limits allowing 0-fee individually
+
+        # Prep for fork with empty blocks to not use invalidateblock directly
```

Comment:
```suggestion
        # Prep for fork with empty blocks
```

I think this should be included in the PR description or commit description rather than in the code. We would not like to refer to a old function we used here which anyway could be easily find if we backtracked to the PR.

Reply:
Agree, I had the same thought! I will remove that for now!